certificate-authority/store/cqldb/config.go

```diff
@@ -0,0 +1,15 @@
+package cqldb
```

Comment:
also here + folder name. This might really differ per db implementing CQL

Reply:
This is a client that you can use with CQL databases.